templating/templating.go

```diff
@@ -43,6 +45,214 @@ type trigger struct {
 	Name string `json:"name"`
 }
 
+func FilterKeys(source template.FuncMap, keys []string) template.FuncMap {
```

Comment:
and let's fix linter and add godoc for this func or make func private better FilterKeys=> filterKeys :)

Reply:
done